gap/semigroups/semitrans.gi

```diff
@@ -741,6 +741,82 @@ function(S)
   return t;
 end);
 
+InstallMethod(WreathProduct,
+"for a transformation monoid and a permutation group",
+[IsTransformationMonoid, IsPermGroup],
+function(M, G)
+  local S;
+  S := AsMonoid(IsTransformationMonoid, G);
+  return WreathProduct(M, S);
+end);
+
+InstallMethod(WreathProduct,
+"for a permutation group and a transformation semigroup",
+[IsPermGroup, IsTransformationSemigroup],
+function(G, S)
+  local M;
+  M := AsMonoid(IsTransformationMonoid, G);
+  return WreathProduct(M, S);
+end);
+
+InstallMethod(WreathProduct,
+"for two transformation semigroups",
```

Comment:
Just one bad string here: you want `"for a transformation monoid and a transformation semigroup"`